clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -146,20 +156,29 @@ def __init__(self,
                     "Reason: {}".format(e.reason))
             self.k8s_namespace = namespace
         else:
-            msg = "Error connecting to Kubernetes cluster. Namespace does not exist"
+            msg = "Error connecting to Kubernetes cluster. Namespace does not exist. You can pass in KubernetesContainerManager(create_namespace_if_not_exists=True) to crate this namespcae"
             logger.error(msg)
             raise ClipperException(msg)
 
+        self.cluster_identifier = "{ns}{cluster}".format(
+            ns=self.k8s_namespace + '-'
+            if self.k8s_namespace != "default" else "",
```

Comment:
This syntax is pretty convoluted. Can you write this as just a normal if-statement outside the format argument (for maintainability reasons)?